services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -106,7 +117,13 @@ const StateUserRoutes = ({
                     path={RoutesRecord.SUBMISSIONS_NEW}
                     element={<NewStateSubmissionForm />}
                 />
-                {showRateSummaryPage && (
+                {showRateEditPage && (
```

Comment:
We should have 2 routes here.
- One route to `RateSummary`, with the feature flag.
```ts
{showRateEditPage && (
  <Route
      path={RoutesRecord.RATES_SUMMARY}
      element={<RateSummary />}
  />
)}
```
- One route to an edit rate Page, can call it something like `RateEdit`
```ts
{showRateEditPage && (
    <Route 
        path={RoutesRecord.RATES_SUMMARY}
        element={<RateEdit />} 
    />
)}
```

The redirect if a rate is unlocked should be done in the `RateSummary` page. Like how we do this for submission summary. You can find this code in `services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx` at line 123.
```ts
    // State users should not see the submission summary page for DRAFT or UNLOCKED, it should redirect them to the edit flow.
    if (
        !(isCMSUser || isAdminUser || isHelpdeskUser || isBusinessOwnerUser) &&
        (submissionStatus === 'DRAFT' || submissionStatus === 'UNLOCKED')
    ) {
        navigate(`/submissions/${id}/edit/type`)
    }
```